src/utils/constants.ts

```diff
@@ -96,13 +98,22 @@ export default {
   LABEL_CONSOLIDATION_DISAGREE: 'Disagree',
   LABEL_CONSOLIDATION_AGREEMENT: 'Agreed',
   LABEL_CONSOLIDATION_DISAGREEMENT: 'Disagreed',
+  LABEL_CREATED: 'created',
```

Comment:
See comment below about "loaded" and breaking up sentences. I don't think it's a good idea.